docs/tools/vscode/README.md

```diff
@@ -33,19 +32,19 @@ If you have ever downloaded and installed a custom font to use in your word proc
 
 We will install required extensions and list optional extensions you can install at the end of the page to add a little pizzazz to your coding experience.
 
-# Required Extensions
+# Required Extensions :id=required-extensions
 
 Install the required extensions:
 
-* [Angular Essentials](https://marketplace.visualstudio.com/items?itemName=johnpapa.angular-essentials) - An all in one extension covering the needs for Angular developers
+- [Angular Essentials](https://marketplace.visualstudio.com/items?itemName=johnpapa.angular-essentials) - An all in one extension covering the needs for Angular developers

```

Comment:
I don't think we're planning on using Angular, if we even finish out this curriculum session. See above notes about tools